.gitignore

```diff
@@ -2,3 +2,10 @@
 
 # Not check in node_modules to a specific arch and os.
 docs/book/node_modules/
+
+# Skip bazel dirs
+bazel-bin
```

Comment:
Can we simply use `bazel-*`?

Reply:
Good call - I did `/bazel-*` so that we won't accidentally match any files from subdirectories.